addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2739,6 +2739,194 @@ options.registry.ReplaceImage = options.Class.extend({
     },
 });
 
+options.registry.ConditionalVisibility = options.Class.extend({
+
+    /**
+     * We clear out the current style tag to make everything visible
+     * @override
+     */
+    init: function () {
+        const style = document.getElementById('wrap') && document.getElementById('wrap').getElementsByTagName('style')[0];
```

Comment:
If you can try using the convention to put "El" at the end of variable containing html elements, I think it is a good idea. So here `styleEl`